test/tsconfig.json

```diff
@@ -1,7 +1,9 @@
 {
```

Comment:
What is this configuration file used for?

Reply:
It defined the folder as its own project that can be referenced by others. It prevents it from being rebuild if nothing changes. Otherwise its rebuild on every change. And by rebuild, I mean rebuild by VSCode, because it's not typically build by `tsc`. Otherwise a change in any ts file of the whole project cause VSCode to have to rebuild this folder too, which slows down Intellisense. Does that make sense?

Reply:
That makes perfect sense, thanks for explaining 👍